src/screens/tenants/Tenants.tsx

```diff
@@ -62,13 +66,21 @@ export default class Tenants extends BaseScreen<Props, State> {
 
   public render() {
     const navigation = this.props.navigation;
-    const { tenants, showAddTenantManuallyDialog, showAddTenantWithQRCode, showAddTenantDialog, showNewTenantAddedDialog } =
-      this.state;
+    const {
+      tenants,
+      showAddTenantManuallyDialog,
+      showAddTenantWithQRCode,
+      showAddTenantDialog,
+      showNewTenantAddedDialog,
+      tenantToBeEdited
+    } = this.state;
     const style = computeTenantStyleSheet();
+    const listItemCommonStyle = computeListItemCommonStyle();
     return (
       <View style={{ flex: 1 }}>
         {showAddTenantDialog && this.renderAddTenantDialog(style)}
         {showNewTenantAddedDialog && this.renderNewTenantAddedDialog(style)}
+        {tenantToBeEdited !== null && this.renderEditTenantDialog(style)}
```

Comment:
Same 'tenantToBeEdited !== null' = 'tenantToBeEdited'

Reply:
I am using the tenant index, so it can be 0
I can rename the state var to be more accurate

Reply:
Yes that would be perfect!